lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_non_nat_dhcp.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+
+import s1ap_types
+from integ_tests.s1aptests import s1ap_wrapper
+from s1ap_utils import MagmadUtil
+
+
+class TestAttachDetachWIthNonNatDhcp(unittest.TestCase):
```

Comment:
Capitalization typo.
```suggestion
class TestAttachDetachWithNonNatDhcp(unittest.TestCase):
```

Reply:
Done.